### PR TITLE
Define ALTERNATE_WORKER_QUEUES

### PR DESCRIPTION
**Pull request by efischer19**
@jibsheet Here's my first pass at this.

I'm going to try a sandbox deploy to test this shortly.

### CODE REVIEW

**Comment by efischer19** [comment]
Needs to land before https://github.com/edx/edx-platform/pull/12689

File: playbooks/roles/apache/templates/lms.j2 [hunk 1]
```diff
@@ -14,6 +14,7 @@ WSGIRestrictEmbedded On
     </Directory>
 
     SetEnv SERVICE_VARIANT lms
+    # Do I need to define ALTERNATE_WORKER_QUEUES here as well?
```
Comment (jibsheet) [inline playbooks/roles/apache/templates/lms.j2]: Shouldn't need this.  I forgot we even have an Apache role.

**Comment by jibsheet** [comment]
Check my logic.  We don't need to touch workers.conf.j2 because workers only care about their queues.  We're ensuring the gunicorn workers can see all queues.  

Need to review the platform PR but this looks fine, and we can probably merge early once we're sure of the names on that PR.

**Comment by efischer19** [comment]
That's the same logic I had - the workers don't need to know about alternate workers, the gunicorn workers do.

I'll get the comment change backed out and merge this after tests pass again.

**Comment by jibsheet** [comment]
retroactive 👍

**Pull request merged**